streams/src/main/java/org/apache/kafka/streams/kstream/internals/SessionKeySerde.java

```diff
@@ -112,11 +112,11 @@ public void close() {
     }
 
 
-    public static long extractEnd(final byte [] binaryKey) {
+    public static long extractEnd(final byte[] binaryKey) {
```

Comment:
Yes, otherwise I couldn't get checkstyle to work. Not sure why it's working for others.